data_structures/src/types.rs

```diff
@@ -69,6 +72,11 @@ impl fmt::Display for Command {
                 }
                 write!(f, ": {}", tx.hash())
             }
+            Command::SuperBlockVote(sbv) => write!(
+                f,
+                "SUPERBLOCK_VOTE #{}: {}",
```

Comment:
Should we include the address that sign the superblock in the display?

Reply:
That may be too verbose.

There is already a log line in chain_manager/mod.rs add_superblock_vote:

                "Received superblock vote from identity {}: #{} {}",

Reply:
the idea is to skip the repeated messages, as I put it in another comment, but we need a way to distinguish when is the same supervote with the same signature, or when the signature is different

Reply:
Done.